instrumentation/runtime-telemetry/runtime-telemetry-java8/library/src/main/java/io/opentelemetry/instrumentation/runtimemetrics/java8/GarbageCollector.java

```diff
@@ -125,11 +132,20 @@ public void handleNotification(Notification notification, Object unused) {
 
       String gcName = notificationInfo.getGcName();
       String gcAction = notificationInfo.getGcAction();
+      String gcCause = notificationInfo.getGcCause();
       double duration = notificationInfo.getGcInfo().getDuration() / MILLIS_PER_S;
-
-      gcDuration.record(
-          duration,
-          Attributes.of(JvmAttributes.JVM_GC_NAME, gcName, JvmAttributes.JVM_GC_ACTION, gcAction));
+      Attributes gcAttributes =
+          enableCaptureGcCause
+              ? Attributes.of(
+                  JvmAttributes.JVM_GC_NAME,
+                  gcName,
+                  JvmAttributes.JVM_GC_ACTION,
+                  gcAction,
+                  AttributeKey.stringKey("jvm.gc.cause"),
```

Comment:
move this to a static final field

Reply:
It was just used once in this class, I didn't define an extra static final field initial.